src/index.html

```diff
@@ -11,12 +11,100 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo__image"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
    <header class="header">
      <a
        href="#"
        class="logo"
      >
        <img
          src="images/logo.png"
          alt="Moyo logo"
          class="logo__image"
        />
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a
              href="#"
              class="nav__link"
            >
              Samsung
            </a>
          </li>
```

It is a good idea to add blank lines between multiline siblings. pls fix everywhere